snakeplane/plugin_factory.py

```diff
@@ -723,6 +723,9 @@ def chunk_ii_close(current_interface: object):
                         return
                     self._build_metadata(plugin)
 
+                if not plugin.initialized:
```

Comment:
Also, it seems like this statement is only going to return `True` if `plugin.initialized` is `None`, because line 721 seems to be the only place where `self._initi_func(plugin)` gets run, and this check happens immediately afterwards. So in that case, I'd expect this to work the same if it were put before line 720. Just trying to get to the root of the issue.

Reply:
I don't think you can put this before line 720 because `initialized` defaults to `False` and if you do this check before line 720 you'd never run the init_func, so the plugin would always terminate early.